packages/js/customer-effort-score/src/style.scss

```diff
@@ -3,6 +3,15 @@
 .woocommerce-customer-effort-score__selection {
 	margin: 1em 0;
 
+	.components-v-stack {
+		display: flex;
```

Comment:
On smaller screens, this still looks a little odd, but I'm not really sure what it was supposed to look like before:

<img width="418" alt="Screen Shot 2022-11-18 at 5 34 39 AM" src="https://user-images.githubusercontent.com/10561050/202717511-c57097da-6466-47cd-9538-3b10007fcfb4.png">

Reply:
Funny, I thought the same and tried centering everything, but it still looked odd. I'll look more into this, but feedback from @jarekmorawski would be helpful.

Reply:
Can we break it into two columns and center them? 🤔

Reply:
I agree that on smaller screens it looks odd, but it is readable and useable. Since this is a code freeze exception, I suggest we merge this fix as is and tackle further improvements for small viewports in the future.

Reply:
We actually ended up favoring @louwie17 's fix which had the icons centered, so this one will be closed. As Matt suggested any further improvements can be captured in another issue.